src/components/portal/portal.tsx

```diff
@@ -67,26 +57,23 @@ export class Portal {
     public position: 'fixed' | 'absolute' = 'absolute';
 
     /**
-     * A unique ID.
+     * @deprecated no longer needed or used, will be removed in a future release
      */
     @Prop({ reflect: true })
     public containerId: string;
```

Comment:
I think we should keep the unique `id`, and use it with `aria-controls` attribute on the trigger element. We have been doing this manually in many components. But it's better if portal takes care of this _if possible_, instead of us remembering and doing it where we use portal.

Anyhow, requiring an `id` is good I would say.

But we have had a problem in the previous implementation of the id. The id used to be put both on `<limel-portal id="uniqueID">` element itself (which wraps the trigger element), and also on the `<div class="limel-portal--container" id="uniqueID">` which is the content of the top layer. Then the trigger element would have the `aria-controls` referring to that `id`, like `<limel-button aria-controls="uniqueID">`. This was wrong. Two elements can't have the same id. Only the "surface" or top-layer should have the id. This is something we can fix now here.

Reply:
> I think we should keep the unique id, and use it with aria-controls attribute on the trigger element. We have been doing this manually in many components. But it's better if portal takes care of this if possible, instead of us remembering and doing it where we use portal.

I want to remove this component entirely eventually so I think it's better to not add more functionality to it now :slightly_smiling_face:

Reply:
I still think we shouldn't deprecate this, as it forces the consumer to do the right thing. See my suggestion: https://github.com/Lundalogik/lime-elements/pull/3088/files#r1713476180

I think when we throw away this component, we can think about how to solve this later, not to make an accessibility bug.

Reply:
But it's not used anywhere in the component :thinking:

Reply:
ok, fair enough. Then what about the last two fixups I made?

Reply:
It looks great! :partying_face: